cfgov/unprocessed/css/molecules/global-search.less

```diff
@@ -0,0 +1,272 @@
+        pointer-events: auto;
+
+        &:focus {
+            outline: 1px dotted @black;
+        }
+
+        .cf-icon:before {
+            content: @cf-icon-search;
+            font-size: unit( 20px / 18px, em );
+        }
+
+        // Show "Search" text in trigger at tablet sizes.
+        .respond-to-min( @bp-sm-min {
+            &-label:before {
+                .webfont-medium();
+                content: 'Search';
+            }
+        } );
+
+        // Only show flyout and close trigger at tablet/mobile sizes.
+        .respond-to-max( @bp-sm-max, {
+            // Increase size of trigger icon at mobile size.
+            font-size: unit( 24px / @base-font-size-px, em );
+
+            &[aria-expanded="true"] {
+                background: @gray-10;
+                border: 1px solid @gray-50;
+                border-bottom: none;
+                color: @black;
+
+                .cf-icon:before {
+                    content: @cf-icon-delete;
+                }
+
+                // Cover up the border edge of the flyout so that it is attached
+                // visually to the trigger button.
+                &:after {
+                    display: block;
+                    height: 1px;
+                    width: 100%;
+
+                    position: absolute;
+                    left: 0;
+                    bottom: -1px;
+
+                    content: '\a0';
+                    background: @gray-10;
+                }
+            }
+        } );
+
+        // Show "Close" text in trigger at tablet sizes.
+        .respond-to-range( @bp-sm-min, @bp-med-min, {
+            &[aria-expanded="true"] {
+                .m-global-search_trigger-label:before {
+                    content: 'Close';
+                }
+            }
+        } );
+
+        .respond-to-min( @bp-med-min {
+            &[aria-expanded="true"] {
+                display: none;
+            }
+        } );
+    }
+
+    &_content {
+        display: none;
+
+        padding-top: unit( @grid_gutter-width / 4 / @base-font-size-px, em );
+
+        &[aria-expanded="true"] {
+            display: block;
+
+            transform: translateY(0);
+            transition: transform 0.25s ease-out;
+            visibility: visible;
+        }
+
+        // Create flyout at table/mobile size.
+        .respond-to-max( @bp-sm-max, {
+            box-sizing: border-box;
+            width: 100%;
+
+            position: absolute;
+            left: 0;
+            // Offset for height of trigger button.
+            top: 60px;
+            z-index: 10;
+
+            background-color: @gray-10;
+            border: 1px solid @gray-50;
+            pointer-events: auto;
+            transform: translateX(100%);
+            transition: transform 0.2s ease-in,
+                        visibility 0s 0.5s;
+            visibility: hidden;
+
+            &-form {
+                padding-top: unit( @grid_gutter-width / @base-font-size-px, em );
```

Comment:
Added in https://github.com/cfpb/cfgov-refresh/commit/dfc1db14aa7a825b0b5b39f8cba24c06d761a2a5